src/RedirectMiddleware.php

```diff
@@ -172,7 +172,7 @@ public function modifyRequest(
         // would do.
         $statusCode = $response->getStatusCode();
         if ($statusCode == 303 ||
-            ($statusCode <= 302 && $request->getBody() && !$options['allow_redirects']['strict'])
```

Comment:
Why was this removed?

Reply:
Am I wrong? :/

Reply:
Nope. You are correct https://github.com/guzzle/psr7/blob/master/src/MessageTrait.php#L126